### DISCUSSION TITLE
Alarm Card Small Buttons

### DISCUSSION THREAD

**Discussion by danbayliss** [home-assistant/frontend#8005] [Dashboard] [answered]
Currently the alarm card has really small numbered buttons. It would be good if this could be changed so the numbers take up the full space of the card

**Comment by spacegaier** [comment]
I made a quick test by removing the max-width constraint from the keypad area. Looks a bit weird since you loose that typical almost square look of the keypad. But might make the buttons a bit easier to hit.

Before:
![grafik](https://user-images.githubusercontent.com/114137/104094777-42d72c00-5293-11eb-9ae1-c0007dd57261.png)

After:
![grafik](https://user-images.githubusercontent.com/114137/104094797-600bfa80-5293-11eb-833b-292528018564.png)

Bigger text inside the buttons might actually work though:
![grafik](https://user-images.githubusercontent.com/114137/104095057-12908d00-5295-11eb-8afb-2abde827b061.png)

**Comment by spacegaier** [comment, marked as answer]
PR created https://github.com/home-assistant/frontend/pull/8120

**Reply by danbayliss** [reply]
Great. I agree that the bigger text looks a lot better. Thank you!
